packages/p2p/src/utils/validations.js

```diff
@@ -3,3 +3,5 @@ export const decimalValidator = v => /^(\d+\.)?\d+$/.test(v);
 export const lengthValidator = v => v.length >= 1 && v.length <= 300;
 
 export const textValidator = v => /^[\p{L}\p{Nd}\s'.,:;()@#+/-]*$/u.test(v);
+
+export const rangeValidator = (input, limit) => input >= limit * -1 && input <= limit;
```

Comment:
```suggestion
export const rangeValidator = (input, limit) => input >= limit * -1 && input <= limit;
```
Add some explanations for this. What is the purpose of this rangeValidator?

Reply:
Thank you for the notice. I will add a deacription.